TorSharp/Tools/LineByLineConfigurer.cs

```diff
@@ -57,14 +57,24 @@ public LineByLineConfigurer(IConfigurationDictionary configurationDictionary, IC
                     }
                 }
 
+                // If the original file exists, remove it before copying over the temporary file.
                 if (File.Exists(path))
                 {
-                    File.Replace(temporaryPath, path, null);
-                }
-                else
-                {
-                    File.Move(temporaryPath, path);
+                    string backupPath = path + ".bak";
+
+                    // If there has already been a backup, just delete the original.
+                    if (File.Exists(backupPath))
+                    {
+                        File.Delete(path);
```

Comment:
FYI I will be making a 1.4.0 release for NuGet when this is merged into `master`.